plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -420,13 +443,16 @@ static bool ffmpeg_hls_mux_start(void *data)
 	stream->total_bytes = 0;
 	obs_output_begin_data_capture(stream->output, 0);
 
-	info("Writing to path '%s'...", stream->path.array);
+	stream->printable_path = path_str;
+	info("Writing to path '%s'...", stream->printable_path);
 
 	return true;
 }
 
 static int deactivate(struct ffmpeg_muxer *stream, int code)
 {
+	obs_service_t *service;
+	const char *temp;
```

Comment:
These two variables are not needed anymore.

Reply:
yep thanks!